hydrophone.go

```diff
@@ -141,8 +143,12 @@ func main() {
 		/*
 		 * hydrophone setup
 		 */
-	store := sc.NewMongoStoreClient(&config.Mongo)
-
+	storage, err := sc.NewStore(&config.Mongo, logger)
```

Comment:
I would keep the same `store` name
```suggestion
	store, err := sc.NewStore(&config.Mongo, logger)
```